sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -140,10 +140,13 @@ void AddCustomerPolicies(HttpPipelinePosition position)
                 else
                 {
                     transport = HttpPipelineTransport.Create(defaultTransportOptions);
+                    return new DisposableHttpPipeline(transport,
```

Comment:
Does DisposableHttpPipeline still need a public constructor if it is created by the builder and set up to use by callers via ClientOptions?

Reply:
good point - marking it internal.